.github/ISSUE_TEMPLATE/join-contributors.md

```diff
@@ -17,3 +17,26 @@ Name | GitHub Handle | PyLadies Chapter | Team Role
 **Why do you want to join the PyLadies Project Communications team?**
 
 /cc @pyladies/communications
+
+**Issues or projects that I'm interested in helping out with**
+
+**Time zone**
+
+**Languages I speak**
+
+|Language | Level |
+| -- | -- 
+|Language 1| Native |
+|Language 2 | Intermediate|
+|Language 3 | Beginner|
+|Language 3 | Beginner |
+
+**Are you comfortable communicating in English?**
+
+**Monthly meetings**: Are you available to attend conference calls once a month? 
```

Comment:
Can we add the slack channel involvement here too?

Reply:
@albendz yep, that's a good idea !